perun-ldapc-ada/src/main/resources/perun-ldapc.xml

```diff
@@ -177,7 +177,7 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 							<value>cz.metacentrum.perun.core.api.Attribute</value>
 						</list>
 					</property>
-					<property name="pattern" value=" set for User:\[(.|\s)*\]" />
+					<property name="pattern" value=" set for User:\[.*\]$" />
```

Comment:
While its not part of changed code I noticed, that you construct regex like this ```Pattern.compile(regexp, Pattern.MULTILINE | Pattern.DOTALL);``` while old LDAPc uses only ```Pattern.DOTALL```.

I don't say, that its wrong, but ``Pattern.DOTALL`` tells that dot matches also all newlines, while ``Pattern.MULTILINE`` tells that ``^`` and ``$`` match on each line of input. And we use ``.*`` to represent any PerunBean object content.

Since you don't match the start of the string (each line) and I believe that present dot matches and consumes everything before end of line is matched, then it probably works.